software/mdtraj.yaml

```diff
@@ -0,0 +1,14 @@
+name: MDTraj
+description: An open library for the analysis of molecular dynamics trajectories
+repository: https://github.com/mdtraj/mdtraj
+languages:
+  - Python
+  - C++
+link: https://github.com/mdtraj/mdtraj
+docs: https://mdtraj.readthedocs.io/en/latest/
+licenses:
+  - GNU LGPLv2.1
```

Comment:
LGPL is fairly common in this space, I recommend adding it to the drop-down. (I happen to not recognize AGPL or 0BSD when I was filling out the form)

Reply:
"LGPL" is probably fine descriptively, fussing with the details here (when you allow the user to input anything) is a black hole